language-extensions/dotnet-core-CSharp/sample/SubModuleCall/SubModuleProject/SubModule/SubModule.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
May I know why this file is needed?

Reply:
The Solution file will help in opening the project in Visual studio. However, as this is a simple project, we can remove it as well. Do you suggest us to remove it?